rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/WorkerSinkTask.java

```diff
@@ -558,8 +558,11 @@ private ConnectRecord convertMessages(MessageExt message) {
         }
 
         // add extension
-        addExtension(properties, record);
-        return record;
+        addExtension(properties, transformedRecord);
+
+        // add msgId to extension
+        transformedRecord.addExtension("MQ-SYS-MSG_ID", message.getMsgId());
```

Comment:
msgId is useful in some cases, such as outputting logs. Because message key is optional. It can be placed in the addExtension method, but the method signature needs to be modified to add the message parameter. I can modify it if necessary.

Reply:
done